lib/challenge_gov/security.ex

```diff
@@ -212,34 +212,57 @@ defmodule ChallengeGov.Security do
   end
 
   # functions to control non-gov challenge manager
-  def intercept_challenge_manager_ng(original_track) do
+  def intercept_user_non_gov(original_track) do
     %{
       original_track
       | originator_role:
-          is_challenge_manager_ng(
+          is_user_non_gov(
             original_track.originator_role,
             original_track.originator_identifier
           )
     }
   end
 
   # check role challenge_manager & .gov or .mil email account
-  def is_challenge_manager_ng(originator_role = "challenge_manager", originator_identifier) do
+  def is_user_non_gov(originator_role = "challenge_manager", originator_identifier) do
     if validate_gov_mil?(originator_identifier) do
       originator_role
     else
       "challenge_manager_ng"
     end
   end
 
-  def is_challenge_manager_ng(originator_role = _, _originator_identifier) do
+  def is_user_non_gov(originator_role = "evaluator", originator_identifier) do
+    if validate_gov_mil?(originator_identifier) do
+      originator_role
+    else
+      "evaluator_ng"
+    end
+  end
+
+  def is_user_non_gov(originator_role = _, _originator_identifier) do
     originator_role
   end
 
   def validate_gov_mil?(email) do
     String.ends_with?(email, [".gov", ".mil"])
   end
 
+  def add_ial_level_to_details(%{originator_id: originator_id} = params)
+      when not is_nil(originator_id) do
+    case ChallengeGov.Accounts.get(originator_id) do
+      %User{ial_level: ial_level} ->
```

Comment:
I wasn't seeing the `ial_level` being logged because this case isn't matching the return value:
```suggestion
      {:ok, %User{ial_level: ial_level}} ->
```